internal/util/json.go

```diff
@@ -83,6 +84,30 @@ func JSONMapFromString(data string) (map[string]any, error) {
 	return target, nil
 }
 
+func MustToJSONMapSlice(data any) []map[string]any {
+	result, err := ToJsonMapSlice(data)
+	if err != nil {
+		log.Fatal("Failed to convert object to a slice of maps.", log.NewAttr("data", data), err)
+	}
+
+	return result
+}
+
+func ToJsonMapSlice(data any) ([]map[string]any, error) {
```

Comment:
I'm not a fan of these.

Something like MustToGenericJSON() makes sense to me since we pass in a slice and expect a slice.
Here we don't even know we have a slice/list coming in.